code/modules/overmap/ship_weaponry/weaponry/grauwolf_probe.dm

```diff
@@ -0,0 +1,108 @@
+/obj/item/ship_ammunition/grauwolf_probe
+	name = "grauwolf sensor probe"
+	desc = "A gun-launched sensor probe, used to scan the area around and relay findings using the datalink."
+	icon = 'icons/obj/guns/ship/grauwolfprobe.dmi'
+	icon_state = "probe_2"
+	overmap_icon_state = "missle_probe"
+	caliber = SHIP_CALIBER_90MM
+	ammunition_behaviour = SHIP_AMMO_BEHAVIOUR_DUMBFIRE
+	impact_type = SHIP_AMMO_IMPACT_PROBE
+	overmap_behaviour = null	// This ammo cannot hit anything
+	projectile_type_override = /obj/item/projectile/ship_ammo/grauwolf_probe
+	burst = 1
+
+/obj/item/ship_ammunition/grauwolf_probe/transfer_to_overmap(var/new_z)
+	var/obj/effect/overmap/start_object = map_sectors["[new_z]"]
+	if(!start_object)
+		return FALSE
+
+	var/obj/effect/overmap/projectile/probe/P = new(null, start_object.x, start_object.y, origin)
+	P.name = name_override ? name_override : name
+	P.desc = desc
+	P.ammunition = src
+	P.target = overmap_target
+	P.range = range
+	if(istype(origin, /obj/effect/overmap/visitable/ship))
+		var/obj/effect/overmap/visitable/ship/S = origin
+		P.dir = S.dir
+	P.icon_state = overmap_icon_state
+	P.speed = 32
+	P.entry_target = entry_point
+	forceMove(P)
+	log_and_message_admins("A projectile ([name]) has entered the Overmap! (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[P.x];Y=[P.y];Z=[P.z]'>JMP</a>)")
+	return TRUE
```

Comment:
Don't really like this copypaste here. Whatever changes here should just be integrated into the parent proc if possible.